zerver/actions/message_send.py

```diff
@@ -1412,6 +1442,7 @@ def check_message(
             "JabberMirror",
         ]
 
+        check_direct_message_policy(realm, sender, user_profiles)
         check_private_message_policy(realm, sender, user_profiles)
```

Comment:
I think we want this to be replacing the existing `check_private_message_policy`, right?